content/README.md

```diff
@@ -0,0 +1,67 @@
+<content>
+
+# README
+
+A guide to show what is required to add a new file with markdown in the `/content` directory.
+
+You can either follow this file or read it on the [developer portal](https://developer.test.digitallibrary.io/README)
+
+Primarly you can use normal markdown syntax. Here is a cheatsheet you can follow: [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+Since we have a design we want to preserve, there are some details that you need to follow.
+
+## Section
+
+A section seperates the rest of the content with a white background and are preferrably used as a top section with a title and some description like this:
+
+```xml
+<section>
+
+# Here you can add a title
+
+Here you can write description
+
+It is optional to add a semi-transparent image like this:
+
+<bottomwrapper>
```

Comment:
Denne komponenten brukes vel nå bare til å plassere et bilde riktig? Har du tenkt at den skal kunne brukes til noe mer eller at det bare er aktuelt å ha et bilde/logo innenfor tagen `<bottomwrapper />`? Hvis tanken er å bare ha et bilde eller en logo så ville jeg renamet den til noe som reflekterer at man kan plassere et bilde/logo i den.

Ser at du lengre nede har sagt: `bottomwrapper: ImageWrapper` så evt da beholde `Image` i navnet så det blir `bottomImageWrapper` eller noe liknende :)

Reply:
Den brukes for å sette et bilde halvveis gjennomsiktig og nederst til venstre. Ganske sært tilfelle, så vurderte nesten å ikke ha med i README. For de eneste bildene som passer er figur lignende bilder som de som er på landingssiden og det tenkte jeg ikke var relevant for Christer. Men siden den finnes har jeg valgt å ta den med.

Siden det er html tags i markdown, så må vi leve med lowercase tags. Setter den til `bottomimagewrapper` for nå. Dette kan vurderes igjen senere @vikre @kstigen